db/migrate/20170305055452_add_default_to_stories.rb

```diff
@@ -0,0 +1,6 @@
+class AddDefaultToStories < ActiveRecord::Migration[5.0]
+  def change
+    change_column :stories, :content, :string, default: " "
+    change_column :stories, :title, :string, default: " "
```

Comment:
we should use an empty string here instead of a string with a space:
`default: ''`
it allows us to look at records to see if they have an empty title or content by using `story.content.present?`